### PR TITLE
feat(sql): added uuid type in postgres decoder

### LINKED ISSUE #1126: [sql] unsupported datatype: UUID
When I sent a query to my Postgres database, this error was returned, but it is common to use this type in Postgres

image
image

### LINKED ISSUE #1280: [sql] - UUID unsuported type

### PR DESCRIPTION
**Pull request by KauanCurbani**
fixes #1126

added UUID type in postgres decoder

### CODE REVIEW

**Comment by FabianLars** [comment]
Thanks for the PR and sorry for the delay, could you also add a changefile similar to https://raw.githubusercontent.com/tauri-apps/plugins-workspace/v2/.changes/sql-column-order.md please?

**Comment by KauanCurbani** [comment]
sorry for the delay

File: .changes/sql-uuid-type.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"sql": patch
+---
+
+### Changes:
+- The `to_json` function in `postgres.rs` has been modified to include `UUID` as a supported type for decoding.
+- In the match statement for `v.type_info().name()`, the type `UUID` has been added to the list of supported types that decode into `JsonValue::String`.
```
Comment (FabianLars) [inline .changes/sql-uuid-type.md]:
```suggestion
Added support for `UUID` columns to the postgres implementation.
```

While _I_ really appreciate descriptions like this. The implementation details typically don't matter to end-users (or from experience, just confuse them), so let's keep this more generic :)

**Review by FabianLars — APPROVED** [review]

**Comment by glib-0** [comment]
I think there's something wrong with this...at least I don't think I'm doing anything wrong? Using PostgreSQL 16, sql plugin 2.0.2. It's nothing complicated - literally just `await db?.select("SELECT * FROM <table> WHERE <column>='<string>'")`. The record I want returns fine, but uuid columns are all always `null`, as though it's tripping up here at the `try_decode()`

**Comment by FabianLars** [comment]
Can you open a new issue with the output of tauri info please?

I never tested this pr so i won't deny your claims.

**Pull request merged**